pyccl/halos/profiles.py

```diff
@@ -290,7 +290,7 @@ def convergence(self, cosmo, r, M, a_lens, a_source, mass_def=None):
             cosmo (:class:`~pyccl.core.Cosmology`): A Cosmology object.
             r (float or array_like): comoving radius in Mpc.
             M (float or array_like): halo mass in units of M_sun.
-            a_lens (float or array_like): scale factor of lens.
+            a_lens (float): scale factor of lens.
             a_source (float or array_like): scale factor of source.
```

Comment:
Can you add to the docs that "If an array, it must have the same shape as `r`"?

Reply:
Done. I was trying to not make the line too long, but I am not sure if I did it correctly.